cylibs/trust/roles/puller.lua

```diff
@@ -77,21 +73,12 @@ function Puller:on_add()
         end), state.AutoTargetMode:on_state_change())
     end
 
-    self.dispose_bag:add(self.target_timer:onTimeChange():addAction(function(_)
```

Comment:
This was added to address the case where someone would pull and then perform several other actions (e.g. buffs) in between pulling and targeting/engaging. Does it break your new multi-pull logic if we add it back?

Reply:
Oh, odd, I didn't delete that intentionally. I'll fix that